app/scenes/Schedule/index.js

```diff
@@ -15,7 +15,7 @@ import moment from 'moment';
 
 import type { ScheduleTalk } from '../../types';
 
-import Splash from 'react-native-smart-splash-screen';
+// import Splash from 'react-native-smart-splash-screen';
```

Comment:
@JedWatson - it was to @cem2ran's - https://github.com/cem2ran/react-conf-app/pull/1